tests/fast-integration/skr-test/provision-skr.js

```diff
@@ -30,13 +30,13 @@ describe('Provision SKR instance', function() {
   });
 
   after('Print Shoot Info', async function() {
-    if (shootInfo != undefined) {
-      console.log(options);
+    // print data out for spinnaker
+    if (options && options.instanceID) {
+      console.log(`SPINNAKER_PROPERTY_INSTANCE_ID=${options.instanceID}`);
```

Comment:
```suggestion
      console.log(`INSTANCE_ID=${options.instanceID}`);
```

Reply:
It needs to have the prefix "SPINNAKER_PROPERTY_" as required by spinnaker.

Reply:
could you elaborate on it why?